packages/terra-form-field/src/Field.scss

```diff
@@ -0,0 +1,80 @@
+@import '~terra-mixins';
+// @import './cerner-mock-theme';
+
+:local {
+  .field {
+    border: 0;
+    margin-bottom: var(--terra-form-field-margin-bottom, 1.428em);
+    max-width: 1020px;
+    padding: 0;
+  }
+
+  // Remove this??
```

Comment:
@neilpfeiffer 
![screen shot 2017-10-30 at 10 28 12 am](https://user-images.githubusercontent.com/14099737/32181341-220598f0-bd62-11e7-8d4c-8f96424ca7a2.png)

Reply:
@yuderekyu Talked with Neil & we removed it. Thanks for pointing it out though! I needed the reminder.